app/components/Checkbox/Checkbox.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react';
+import CustomCheckbox from './Checkbox';
+import userEvent from '@testing-library/user-event';
+
+describe('CustomCheckbox', () => {
+    const label = 'Label';
+    it('should display checkbox', async () => {
+        render(<CustomCheckbox label={label} />);
+        const checkbox = screen.getByRole('checkbox');
+        expect(checkbox).toBeInTheDocument();
+        expect(screen.queryByRole('checkbox')).not.toBeChecked();
+        await userEvent.click(screen.getByRole('checkbox'));
+        expect(screen.getByRole('checkbox')).toBeChecked();
+    });
+    it('should check and uncheck the checkbox', async () => {
+        await userEvent.click(screen.getByRole('checkbox'));
+        expect(screen.getByRole('checkbox')).toBeChecked();
+        await userEvent.click(screen.getByRole('checkbox'));
+        expect(screen.queryByRole('checkbox')).not.toBeChecked();
+
+    });
+    it('should not toggle disabled checkbox', async () => {
+        const disabled = true;
+        render(<CustomCheckbox disabled={disabled} label={label} />);
```

Comment:
```suggestion
        render(<CustomCheckbox disabled label={label} />);
```
I am not sure but it seems like it might work without `const disabled = true;` 
And if it is correct so do like this in next test case too